examples/CantileverDistributedLoad/cantilever_nonconservative_distributed_load.py

```diff
@@ -0,0 +1,288 @@
+                    system.compute_position_center_of_mass()
+                )
+                self.callback_params["velocity_magnitude"].append(
+                    (
+                        stretchable_rod.velocity_collection[-1][0] ** 2
+                        + stretchable_rod.velocity_collection[-1][1] ** 2
+                        + stretchable_rod.velocity_collection[-1][2] ** 2
+                    )
+                    ** 0.5
+                )
+
+    recorded_history = ea.defaultdict(list)
+
+    stretch_sim.collect_diagnostics(stretchable_rod).using(
+        AxialStretchingCallBack, step_skip=200, callback_params=recorded_history
+    )
+
+    stretch_sim.finalize()
+    timestepper = ea.PositionVerlet()
+    # timestepper = PEFRL()
+
+    total_steps = int(final_time / dt)
+    print(stretch_sim)
+    print("Total steps", total_steps)
+    ea.integrate(timestepper, stretch_sim, final_time, total_steps)
+
+    if PLOT_FIGURE_Equilibrium:
+
+        plt.plot(
+            recorded_history["time"],
+            recorded_history["velocity_magnitude"],
+            lw=1.0,
+            label="velocity_magnitude",
+        )
+        plt.xlabel("Time (s)")
+        plt.ylabel("Speed (m/s)")
+        plt.title("Tip Position and Speed vs. Time")
+        plt.legend()
+        plt.grid()
+        plt.show()
+
+    rendering_fps = 30
+    if Animation:
+        plot_video_with_surface(
+            [recorded_history],
+            video_name="cantilever_Non-conservative_distributed_load.mp4",
+            fps=rendering_fps,
+            step=1,
+            # The following parameters are optional
+            x_limits=(-0.0, 0.5),  # Set bounds on x-axis
+            y_limits=(-0.5, 0.0),  # Set bounds on y-axis
+            z_limits=(-0.0, 0.5),  # Set bounds on z-axis
+            dpi=100,  # Set the quality of the image
+            vis3D=True,  # Turn on 3D visualization
+            vis2D=False,  # Turn on projected (2D) visualization
+        )
+
+    pos = stretchable_rod.position_collection.view()
+
+    tip_position = Find_Tip_Position(stretchable_rod, n_elem)
+    relative_tip_position = np.zeros((2,))
+
+    relative_tip_position[0] = tip_position[0] / base_length
+    relative_tip_position[1] = -tip_position[1] / base_length
+
+    print(relative_tip_position)
+    return relative_tip_position
+
+
+final_time = 10
+# setting up test params
+n_elem = 100
+start = np.zeros((3,))
+direction = np.array([1.0, 0.0, 0.0])
+normal = np.array([0.0, 1.0, 0.0])
+base_length = 0.5
+base_radius = 0.01 / (np.pi ** (1 / 2))
+base_area = np.pi * base_radius**2
+density = 1000
+dimentionless_varible = 15
+youngs_modulus = 1.2e7
+# For shear modulus of 1e4, nu is 99!
+poisson_ratio = 0
+shear_modulus = youngs_modulus / (2 * (poisson_ratio + 1.0))
+I = (0.01**4) / 12
+
+
+cantilever_subjected_to_a_nonconservative_load(
+    n_elem, base_length, base_radius, youngs_modulus, -15, True, False
+)
+
+x_tip_paper = [
```

Comment:
Same as the other file. Maybe load data from a file, and do not run the same simulation more than once.